index.js

```diff
@@ -138,3 +138,41 @@ newSightingForm.addEventListener('submit', function (event) {
 /************************** EVENTS PART 2 JS MINI CHALLENGE ******************************/
 
 
+
+const animalsUl = document.querySelector("#animals")
+animalsUl.addEventListener('click', function(event){
+    if(event.target.matches("button.like-button")){
+        const likesPtag = event.target.previousElementSibling
+        const likes = parseInt(likesPtag.textContent) + 1
+        likesPtag.textContent = `${likes} Likes`
+      
+    } 
```

Comment:
watch the syntax here. When you have an if/else statement it should look like this
```
if (condition) {
 // logic
} else if (condition) {
 //logic
}
```

`} else if (event.target.matches('button.delete-button')) {`